new-lamassu-admin/src/components/date-range-picker/Calendar.js

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react'
+import moment from 'moment'
+import { toInteger } from 'lodash/fp'
+import { makeStyles } from '@material-ui/core/styles'
+import Tile from './Tile'
+import { ReactComponent as Arrow } from '../../styling/icons/arrow/month_change.svg'
+import { primaryColor, zircon, fontSecondary } from '../../styling/variables'
+import typographyStyles from '../typography/styles'
+const { label2 } = typographyStyles
+
+const styles = {
+  navbar: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    padding: [[15, 15]],
+    color: primaryColor,
+    '& button': {
+      display: 'flex',
+      alignItems: 'center',
+      padding: 0,
+      border: 'none',
+      backgroundColor: zircon,
+      cursor: 'pointer',
+      borderRadius: '50%',
+      width: 20,
+      height: 20,
+      position: 'relative',
+      overflow: 'hidden',
+      '& svg': {
+        position: 'absolute',
+        left: 0
+      }
+    },
+    fontFamily: fontSecondary,
+    fontSize: 16,
+    fontWeight: 500
```

Comment:
this should be extended from some predefined typography

Reply:
The reasoning behind this and https://github.com/lamassu/lamassu-server/pull/328#discussion_r345377519 was that they are both specific things inside this container that shouldn't really be changed globally (more specifically the font size).